components/ChunksPicker.js

```diff
@@ -62,13 +68,16 @@ const createPostReqOptions = (obj) => {
     body: JSON.stringify(obj),
   };
 };
+
 const ChunksPicker = ({ entryFile, className }) => {
   const classes = useStyles();
-  // loads all descendent chunks
+
+  const [skipImportWithoutChunkName, setSkipImportWithoutChunkName] = useState(false);
+
   const loadAllDescendantChunks = useCallback(
     (filepath) =>
       new Promise((resolve, reject) => {
-        fetch('api/chunks', createPostReqOptions({ path: filepath, getDescendant: true }))
+        fetch('api/chunks', createPostReqOptions({ path: filepath, getDescendant: true, skipImportWithoutChunkName:skipImportWithoutChunkName }))
```

Comment:
no need for alias 
createPostReqOptions({ path: filepath, getDescendant: true, skipImportWithoutChunkName }))